puppetboard/templates/report.html

```diff
@@ -60,7 +37,7 @@ <h1>Summary</h1>
 
 <h1 id="logs">Logs</h1>
 {{ macros.checkbox_toggle("friendly", current_show_error_as, "friendly", "raw", "Friendly error messages") }}
-<table id="logs_table" class='ui basic compact table'>
+<table id="logs_table" class='ui very basic compact striped table'>
```

Comment:
Nit: "striped" styling is really nice, makes reading a table so much easier! Perhaps you could apply this styling to all other tables, especially: nodes in overview, failures, nodes, facts, inventory and query result (in a table form)? Or the other way around: NOT apply it in this PR, but do apply it in a separate one?

Reply:
I rollback this line. I think it would be better in a separate PR with UI unification.